Libraries/ART/ARTShape.m

```diff
@@ -28,7 +28,7 @@ - (void)dealloc
 
 - (void)renderLayerTo:(CGContextRef)context
 {
-  if ((!self.fill && !self.stroke) || !self.d) {
+  if ((!self.fill && !self.stroke && !self.gradientStroke.count) || !self.d) {
```

Comment:
Why do noting if `self.gradientStroke.count` is 0?

What if `self.gradientStroke` is undefined or null? Will this crash?